docs/UserGuide.md

```diff
@@ -402,7 +402,7 @@ Scotland, Edinburgh
 ### `null` _(field)_
 
 ```javascript
-{ "field": "price", "is": "null" }
+{ "field": "price", "null" }
```

Comment:
@cdowding-sl How is `null` being handled in the new profile formatting? Is it a key without a value (as shown here) or an alternative?

Reply:
@cdowding-sl  think we agreed on this in yesterday's meeting......

 " { "field": "foo", "isNull": "true" }"